ooiservices/app/uframe/controller.py

```diff
@@ -464,7 +464,7 @@ def get_uframe_stream_metadata_times(ref):
 
 #@cache.memoize(timeout=3600)
 #DEPRECATED
-def get_uframe_stream_contents(mooring, platform, instrument, stream_type, stream, start_time, end_time, dpa_flag, provenance, annotations):
+def get_uframe_stream_contents(mooring, platform, instrument, stream_type, stream, start_time, end_time, dpa_flag, provenance='false', annotations='false'):
```

Comment:
Nope, for those I pass it directly in from the user, but for csv I don't give them the option so we need to set defaults. The default is false and will only be used by csv.